core/app/[locale]/(default)/compare/page.tsx

```diff
@@ -57,21 +57,28 @@ const getProducts = cache(async (productIds: number[] = []): Promise<CompareCard
   }));
 });
 
-export async function generateMetadata(): Promise<Metadata> {
-  const t = await getTranslations('Compare');
-
-  return {
-    title: t('title'),
-  };
-}
-
```

Comment:
This diff is a bit weird because I moved the `Props` interface to be before `generateMetadata` to be consistent with other files. GitHub thinks the entire `generateMetadata` function was remove, but it's down below with the `params.locale` change that has been present in other files in this PR.